components/network/openssh/patches/0033-test.patch

```diff
@@ -0,0 +1,8 @@
+diff -wpruN '--exclude=*.orig' a~/regress/cfgparse.sh a/regress/cfgparse.sh
+--- a~/regress/cfgparse.sh	1970-01-01 00:00:00
++++ a/regress/cfgparse.sh	1970-01-01 00:00:00
+@@ -73,3 +73,4 @@ EOD
+ 
+ # cleanup
+ rm -f $OBJ/sshd_config.[012]
++exit 0
```

Comment:
Why is it needed?

Reply:
It's not really clear why it's needed, I dropped it.